main-01.py

```diff
@@ -23,9 +23,8 @@ def get_all_song_keys(all_song_keys):
         if(sufname=="mp3"):
           thismp3=name+".mp3"
           all_song_keys.append(thismp3)
-        else: pass
```

Comment:
You do not need to do `else: pass`.  Not having the statement has the same effect.  Having it is a little confusing.